src/MovieInfo/MovieInfo.js

```diff
@@ -1,55 +1,71 @@
 import React, { Component } from 'react';
+import Error from '../Error/Error';
 import './MovieInfo.css';
 
 class MovieInfo extends Component {
   constructor(props) {
     super(props);
     this.state = {
       currentMovie: {},
-      movieId: this.props.currentMovieId
+      movieId: this.props.currentMovieId,
+      error: ''
     }
   }
 
   componentDidMount() {
-    console.log(this.state.movieId)
     fetch(`https://rancid-tomatillos.herokuapp.com/api/v2/movies/${this.state.movieId}`)
-    .then(response => response.json())
-    .then(data => {
-      this.setState({ currentMovie: {...data.movie} })
-      console.log(this.state.currentMovie)
-    })
+      .then(response => response.json())
+      .then(data => {
+        this.setState({ currentMovie: { ...data.movie } })
+      })
+      .catch(error => this.setState({ error: error.message }))
+  }
+
+  formatCosts = (number) => {
+    if (number === 0) {
+      return `Not Reported`;
+    }
+    const trimmedTotal = number.toFixed(2);
+    console.log(trimmedTotal);
+    let totalFormatted = trimmedTotal.toString().split('.');
+    totalFormatted[0] = totalFormatted[0].replace(/\B(?=(\d{3})+(?!\d))/g, ',');
+    return '$' + totalFormatted.join('.');
   }
 
   render() {
-    console.log(this.state.currentMovie)
     const { poster_path, backdrop_path, release_date, overview, average_rating, genres, budget, revenue, tagline, runtime, title, id } = this.state.currentMovie
-    if(!this.state.currentMovie.title) {
-      return(
-        <h1>Loading</h1>
-      )
-    }
+
+
+
     return (
-      <section className="movie-info-section">
-        <h1 className='movie-info-title'>{title}</h1>
-        <div className='movie-info-container'>
-          <article className="movie-info">
-            <h2 className='list-title'>Movie Info</h2>
-            <ul className='movie-info-list'>
-              <li><b>Genres:</b> {genres.join(', ')}</li>
-              <li><b>Release Date:</b> {release_date}</li>
-              <li><b>Runtime:</b> {runtime} minutes</li>
-              <li><b>Budget:</b> ${budget}</li>
-              <li><b>Revenue:</b> ${revenue}</li>
-            </ul>
-          </article>
-          <article className="movie-description">
-            <img className="backdrop-image" alt={title + 'poster'} src={backdrop_path} />
-            <h2 className='movie-info-title'>{tagline}</h2>
-            <h3 className='movie-info-rating'>Rating: {average_rating}</h3>
-            <p className='movie-info-overview'>{overview}</p>
-          </article>
-        </div>
-      </section>
+      <>
+        {this.state.error && <Error />}
+        {!this.state.currentMovie.title && !this.state.error && <h1>Loading...</h1>
```

Comment:
Nice job getting this moved into the return instead of floating outside as if statements